ts/nni_manager/common/utils.ts

```diff
@@ -213,7 +212,7 @@ function cleanupUnitTest(): void {
     Container.restore(TrainingService);
     Container.restore(DataStore);
     Container.restore(Database);
-    Container.restore(ExperimentStartupInfo);
+    setExperimentStartupInfo(true, 'unittest', 8080, 'unittest', undefined, logLevel);
```

Comment:
Why use 'unittest' as experiment id here?

Reply:
The original logic was snapshot ExperimentStartupInfo in `prepareUnitTest` and restore it here with IOC's utility.
To share ExperimentStartupInfo with 3rd-party training service, it is no longer an IOC container and has to be manually restored.
This line is copied from `prepareUnitTest` to restore the same state.